index.html

```diff
@@ -0,0 +1,226 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio Mfadhili Gitau</title>
+
+    <!-- Google Fonts  -->
+
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;500;600;700&display=swap" rel="stylesheet">
+
+    <!-- CSS file  -->
+    
+    <link rel="stylesheet" href="./css/main.css">
+</head>
+<body>
+    <header>
+       <div class="welcome">
+            <p class="welcome__text">Welcome</p>
+        </div> 
+        <img src="images/Normal Button/Tertiary/Enabled.svg" alt="menu" class="menu">
+
+        <nav class="nav">
+        </nav>
+        
+    </header>
+
+    <main class="main">
+
+        <!-- Main Page section -->
+
+        <section class="headline">
+            <h1 class="greetings">Hello there.I'm Mfadhili</h1>
+            <h3 class="intro">Your Software developer!</h3>
+            <p class="info__text">
+                I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                don’t hestiate to contact me.
+            </p>
+
+            <ul class="social__media">
+                <li class="handles"><a href="http://"><img src="./images/social media/github.svg" alt="github" srcset=""></a></li>
+                <li class="handles"><a href="http://"><img src="./images/social media/Linkedin icon.svg" alt="Linkedin" srcset=""></a></li>
+                <li class="handles"><a href="http://"><img src="./images/social media/twitter.svg" alt="Twitter" srcset=""></a></li>
+                <li class="handles"><a href="http://"><img src="./images/social media/medium.svg" alt="twitter" srcset=""></a></li>
+            </ul>
+        </section>
+
+        <!-- Recent Works section -->
+
+        <section class="recent__works">
+            <section class="work__post">
+                <h2 class="works__heading">My Recent Works</h2>
+                <!-- Project card 1 -->
+                <article class="project1">
+                    <div class="container">
+                        <div class="card">
+                            <div class="card__img">
+                                <img src="/images/Card/Img Plaholder.svg" alt="Placeholder">
```

Comment:
[OPTIONAL] 
- [ ] Kindly make sure you use the correct path for your images as you can see they are not displaying correctly.
`hint: you can remove the space between the name you gave your images`